### ISSUE TITLE
Initialization of wind with multiple domans

### ISSUE THREAD

**Issue by kslong** [sirocco-rt/sirocco#303]
Currently there is an issue in how winds are initialized (and how data for them is read in).  Here is one example of this.  Currently the initial temperature of the wind is set through the variable geo.twind --> geo.twind_init (recently), and all valid plasma cells are initialized to this temperature. But if you have a two component wind; you will be asked for twind_init twice and the last one will be used.  There is a placeholder zdom[].twind_init (which I have just commented out), but it is not used.

What needs to be done is to decide whether for a multi-component wind, we want to allow for the possibility for each component of the wind to be initialized to  a separate value, and then make sure that the plasma cells are all filed using variables in zdom.  This should not be hard.

**Comment by jhmatthews** [comment]
I note that there is a similar issue regarding the maximum radius of the wind, which Knox has documented in get_wind_params() in setup.c (see also #305 )

```There is something of a philosophical problem that needs to be worked
   * out with geo.rmax and zdom[ndom].rmax for the general case of winds.  Suppose
   * we wish to create, say a spherical outflow with two domains one going from 
   * r1 to r2 and the other going from r2 to r3.  Then we want to keep geo.rmax which is 
   * intended to be the distance beyond which photons are moving through free space separate
   * from the values in the wind zones.  Right now we are setting the outer limit of each
   * wind to be geo.rmax regardless, in routines like get_stellar_wind_params and get_sv_wind
   * This is not what we want.  What should happen is that for each componetn where it is
   * relevant we should ask for the outer edge of the domain and then at the end we should determine
   * what geo.rmax should be set to.  There are some cases, e.g. get_hydro_wind where one should not
   * need to ask the question about rmax, but others where it is necessary
```

**Comment by kslong** [comment]
The problem for tinit was fixed at some point in the past, but whoever fixed it (likely me) did not document here.

Now, when an answer to the question Wind.t.init is asked, it is filling zdom[ndom].tinit.  The there is no variable geo.twind or geo.twind_init.
I have verified that initial temperatures are defined by domain.

There is a reference above to a similar problem with geo.rmax, but this was what was fixed as a result of issue #305.

Specicially, we now read
rddoub ("Wind.radmax(cm)", &zdom[ndom].rmax);
into a domain variable.  

I have verified that this also has the effect wanted, namely that this is used to limit a particular domain.  geo.rmax still exists, and is needed.  It defines when a photon has flown out of all of the domains.  

I have inspected the places where geo.rmax is still used and they all seem to conform to this definition.

So this issue is now closed.
